volatility/framework/plugins/windows/netscan.py

```diff
@@ -28,8 +28,8 @@ def get_requirements(cls):
                                                      description = 'Memory layer for the kernel',
                                                      architectures = ["Intel32", "Intel64"]),
             requirements.SymbolTableRequirement(name = "nt_symbols", description = "Windows kernel symbols"),
-            requirements.PluginRequirement(name = 'poolscanner', plugin = poolscanner.PoolScanner, version = (1, 0, 0)),
-            requirements.PluginRequirement(name = 'info', plugin = info.Info, version= (1, 0, 0)),
+            requirements.VersionRequirement(name='poolscanner', component=poolscanner.PoolScanner, version=(1, 0, 0)),
```

Comment:
It would be handy if you could run `yapf` over your code before submitting it please.  Otherwise it tends to mean we have a lot of "running `yapf`" commits to get the formatting consistent.

Reply:
Sure! I didn't know that we should use it 5:)
I ran it with the .style.yapf file and it changed a lot of files, do you want me to run it only on the files i changed? or commit all the file changes?